erp-data-exchange-specification.md

```diff
@@ -261,6 +262,29 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
 insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
 ```
 
+### responsibilityChannels.csv
+
+The `responsibilityChannels.csv` file describes the relation between an agent and a resource channel
+
```

Comment:
Maybe we should clarify what a `resource channel` is. This term isn't used here before.

Reply:
Other that this LGTM